src/app/modules/main/chat_section/chat_content/view.nim

```diff
@@ -1,6 +1,6 @@
 import NimQml
 import ../../../shared_models/message_model as pinned_msg_model
-import ../../../../../app_service/service/contacts/dto/contacts as contacts_dto
+import ../../../../../app_service/service/chat/dto/chat as chat_dto
```

Comment:
Usually views shouldn't have access to the dto under our architecture, but it was already accessing one and TBH, it's an annoying rule, so I'll let it slide lol